src/preferences.jl

```diff
@@ -0,0 +1,19 @@
+@static if VERSION >= v"1.6"
+    using Preferences
+end
+
+@static if VERSION >= v"1.6"
+    const int2float_type = @load_preference("int2float", "Float64")
+else
+    const int2float_type = "Float32"
+end
+
+function set_int2float!(int2float_type::String)
+    if !(int2float_type in ("Float64", "Float32"))
+        throw(ArgumentError("Invalid int2float type: \"$(int2float_type)\""))
+    end
+
+    # Set it in our runtime values, as well as saving it to disk
+    @set_preferences!("int2float" => int2float_type)
+    @info("New int2float type set; restart your Julia session for this change to take effect!")
```

Comment:
Maybe you know already, but one can set preferences before loading a package by specifying the package via its UUID.

Reply:
Thanks, I've used this in running the tests, anywhere else where this would be helpful?